docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -80,9 +84,15 @@
     </ul>
     <hr/>
     <ul class="nav" id="secondary-nav">
-        <li {{ if $in_api_section }} class="active" {{ end }}>
-            <a class="docs-home" href="/docs/apis/" {{ if $in_api_section }}
-                aria-label="Current Page: API Overview" {{ end }}>API Overview</a>
+        <li>
+            <select name="API version" id="api-version" onchange="window.location.href = this.value;">
+                {{ $selectedApiVersion := trim $current.RelPermalink "/" }}
+                {{ $selectedApiVersion = index (split $selectedApiVersion "/") 2 }}
```

Comment:
Nit: we do this same splitting logic in a couple of places. Is there any way for us to extract this into a sharable helper of some kind? I don't know how easy this is in Hugo, so it may not be worth the effort if it's complicated to do.

At the very least, we should document what we expect the URL to look like and what the magic number `2` here is for.

Reply:
The easiest way to check the current viewing version is check the url, which includes the version number in. Adding comments here.

Reply:
If possible, it would still be nice to create a single helper function that all of the places can call. But if that's not possible, docs would be great.

Reply:
Seems like it's better to have a version matching helper function, then if there is version number naming rule change, it will only need to update one place..